wp-web-push-admin.php

```diff
@@ -26,34 +26,34 @@ public function options() {
       } else if ($_POST['webpush_title'] === 'custom') {
         update_option('webpush_title', $_POST['webpush_title_custom']);
       } else {
-        wp_die('Wrong value for the Notification Title');
+        wp_die(__('Wrong value for the Notification Title'));
       }
 ?>
-<div class="updated"><p><strong>Settings saved.</strong></p></div>
+<div class="updated"><p><strong><?php echo __('Settings saved'); ?></strong></p></div>
```

Comment:
I know, I thought I'd try to be consistent;  I can change to `_e` though  *shrugs*